admin/views/school/_pupil-form.ejs

```diff
@@ -11,14 +11,14 @@ if (formData._id) {
 
 <div class="grid-row">
     <div class="column-two-thirds">
+
         <% if (err.hasError()) { %>
         <div class="error-summary" role="group" aria-labelledby="error-summary-heading-1" tabindex="-1">
             <h1 class="heading-medium error-summary-heading" id="error-summary-heading-1">
-                Check the pupil’s details
+                Check the pupil's details
```

Comment:
I suggest we use the correct curly typographic quotes rather than straight ones for page content.  Please revert.